cnd/src/ethereum.rs

```diff
@@ -1,22 +1,140 @@
 #![warn(unused_extern_crates, missing_debug_implementations, rust_2018_idioms)]
 #![forbid(unsafe_code)]
 
+use bitcoin::hashes::core::{fmt::Formatter, str::FromStr};
+use core::fmt;
```

Comment:
This one is confusing also.  There exists `std::fmt::LowerHex`, is that not the same as the usage below of `fmt::LowerHex`?

Reply:
Likely, @da-kami just went with importing stuff through CLion and not writing those himself. Good catch :)

It is unfortunate that it does that :(

Reply:
Auto-importing with rust-analyzer in emacs can also cause similar issues :(